src/components/victory-scatter/victory-scatter.jsx

```diff
@@ -73,7 +73,12 @@ export default class VictoryScatter extends React.Component {
     height: CustomPropTypes.nonNegative,
     /**
      * The labelComponent prop takes in an entire, HTML-complete label component
-     * which will be used to create labels for scatter to use
+     * which will be used to create labels for each point in the scatter plot. The new element
```

Comment:
@cpb this all sounds correct to me. Thanks for improving the docs along the way :)